src/components/MewTable/MewTable.vue

```diff
@@ -364,19 +380,33 @@ export default {
     indexedItems() {
       return this.tableData.map((item, index) => ({
         id: index,
-        ...item,
+        ...item
       }));
-    },
+    }
   },
   watch: {
     selectedValues: {
-      handler: function(newVal) {
+      handler: function (newVal) {
         this.selected = newVal;
       },
-      deep: true,
-    },
+      deep: true
+    }
   },
   methods: {
+    validateCoin(address) {
```

Comment:
This should be done in v6. not in the component.

Reply:
Removed token validation function and modified MewTable to get token type from item.coinType prop.